### PR TITLE
add cbinds for merge (fix #52)

### LINKED ISSUE #52: c_bind for Native Merge Rollup Circuit

### PR DESCRIPTION
**Pull request by rahul-kothari**
# Description

Fixes #52 

# Checklist:

- [x] I have reviewed my diff in github, line by line.
- [x] Every change is related to the PR description.
- [x] I have [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) this pull request to the issue(s) that it resolves.
- [x] There are no unexpected formatting changes, superfluous debug logs, or commented-out code.
- [x] There are no circuit changes, OR specifications in `/specs` have been updated.
- [x] There are no circuit changes, OR a cryptographer has been assigned for review.
- [x] I've updated any terraform that needs updating (e.g. environment variables) for deployment.
- [x] The branch has been rebased against the head of its merge target.
- [ ] I'm happy for the PR to be merged at the reviewer's next convenience.
- [ ] New functions, classes, etc. have been documented according to the doxygen comment format. Classes and structs must have `@brief` describing the intended functionality.
- [x] If existing code has been modified, such documentation has been added or updated.

> **Note**
> If you are updating the submodule, please make sure you do it in its own _special_ PR and avoid making changes to the submodule as a part of other PRs.
> To update a submodule, you can run the following commands:
> ```console
> $ git submodule update --recursive
> ```
> Alternatively, you can select a particular commit in `barretenberg/aztec3` that you wish to point to:
> ```console
> $ cd barretenberg
> $ git pull origin aztec3        # This will point to the latest commit in `barretenberg/aztec3`
> $ git checkout <commit_hash>    # Use this if you wish to point to a particular commit.
> $ cd ..
> $ git add . && git commit -m <commit_msg>
> $ git push
> ```

### CODE REVIEW

File: cpp/src/aztec3/circuits/rollup/merge/.test.cpp [hunk 1]
```diff
@@ -14,7 +15,49 @@ using DummyComposer = aztec3::utils::DummyComposer;
 } // namespace
 namespace aztec3::circuits::rollup::merge::native_merge_rollup_circuit {
 
-class merge_rollup_tests : public ::testing::Test {};
+class merge_rollup_tests : public ::testing::Test {
```
Comment (ludamad) [inline cpp/src/aztec3/circuits/rollup/merge/.test.cpp:18]: I know you're just editing this, but I thought we were going for CamelCase classes

Comment (ludamad) [inline cpp/src/aztec3/circuits/rollup/merge/.test.cpp:18]: I know you're just editing this, but I thought we were going for CamelCase classes

Comment (ludamad) [reply cpp/src/aztec3/circuits/rollup/merge/.test.cpp:18]: Ah I see, it's technically a test suite, I retract my comment I suppose

**Review by ludamad — APPROVED** [review]

**Review by dbanks12 — APPROVED** [review]
Lgtm

**Pull request merged**